Services/Interfaces/web_interface/models/profiles.py

```diff
@@ -19,6 +19,9 @@
 import octobot_tentacles_manager.api as tentacles_manager_api
 
 
+COPY_TRADING_PROFILE_ID = "copy_trading"
```

Comment:
I don't think we should store data here. I think the config would be a better place for it

Reply:
done using https://github.com/Drakkar-Software/OctoBot-Commons/pull/235

Reply:
now taking a param in API